automation/src/test/scala/org/broadinstitute/dsde/firecloud/config/UserPool.scala

```diff
@@ -78,5 +78,14 @@ object UserPool {
     NotebooksWhitelisted.getRandomCredentials(n)
   }
 
+  /**
+    *
+    * Chooses a campaign manager
+    */
+  def chooseCampaignManager: Credentials = chooseCampaignManagers(1).head
+
+  def chooseCampaignManagers(n: Int): Seq[Credentials] = {
```

Comment:
Why are we randomizing campaign managers? I know other tests do the same but it's not clear to my why. Is it to prevent race conditions or something?

Reply:
No other reason than consistency. I think it is a desired feature that test-writers will want to just choose a category of user and go instead of trying to reason about a specific user.